sdk/metric/manual_reader.go

```diff
@@ -0,0 +1,86 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metric // import "go.opentelemetry.io/otel/sdk/metric"
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"go.opentelemetry.io/otel/sdk/metric/export"
+)
+
+// ManualReader is a a simple Reader that allows an application to
+// read metrics on demand.  It simply stores the Producer interface
+// provided through registration.  Flush and Shutdown are no-ops.
```

Comment:
We no longer need to mention the unexported `Producer` interface. Additionally, the `Flush` and `Shutdown` methods are documented themselves, no need to mention them here.

```suggestion
// read metrics on demand.
```